scale-ci-watcher.sh

```diff
@@ -117,8 +120,18 @@ function update_scale_ci_jobs() {
 			echo "--------------------------------------------------"
 			echo "INFO: Found $template"
 			echo -e "--------------------------------------------------\n"
-                	jenkins-jobs --conf "$JJB_CONFIG_PATH" update "$template"
-                	if [[ $? != 0 ]]; then
+			retries=0
+         		for (( iter=1; iter<=$watcher_retries; iter++ ))
```

Comment:
Not that I'm hoping for one, but does there need to be any type of wait/sleep interval between each attempt?

Reply:
Makes sense, added a sleep interval between each attempt @dry923. PTAL.